Meadow_DotNet_SDK/readme.md

```diff
@@ -33,7 +33,7 @@ Additionally, at the top of the solution is a `Directory.Build.props` file which
 </Project>
 ```
 
-One important thing to note, there is a bug in Visual Studio (Windows) that requires the `TargetFramework` to be set on all `.props` files, even though it's set here.  
```

Comment:
NIT: Any chance we can fix these white spaces, as it shows more changes than necessary. There seem to be 3 of them.

Reply:
Technically removing them was a fix. It's extra trailing whitespace that VS Code removed automatically for me. I can put them back, I guess.

Reply:
Done

Reply:
Yes, I see, but it just seemed to detract from the real change. But like I said it was a nit, so we could have just left it.